docs/src/assets/js/themes.js

```diff
@@ -16,7 +16,8 @@
             dark_theme_toggle = document.getElementById('dark-theme-toggle');
 
         // get any previously-chosen themes
-        var theme = window.localStorage.getItem("theme");
+        var localStorageTheme = window.localStorage.getItem("theme") 
+        var theme = localStorageTheme ? localStorageTheme : window.matchMedia('(prefers-color-scheme: dark)').matches? 'dark' : 'light';
```

Comment:
```suggestion
        var theme = document.documentElement.getAttribute('data-theme');
```

Since we are already calculating and applying the initial theme [here](https://github.com/eslint/eslint/blob/eee43067f635c0ec3b61e416f47849029d12268d/docs/src/_includes/layouts/base.html#L65-L73), would it make sense to get that value to avoid duplicating logic?

Reply:
Yep we could do it too! thanks :)